frameworks/hdfs/tests/test_universe.py

```diff
@@ -0,0 +1,31 @@
+import collections
```

Comment:
@rathohit311356 you were going to delete this test?

Reply:
Apologies , i missed to delete from hdfs , done now